### ISSUE TITLE
Emoji bigrams

### ISSUE THREAD

**Issue by nowherenearithaca** [mroth/emojitrack-feeder#15]
Don't know if would be interesting or not yet, but was wondering if you had considered storing emoji bigrams. Seems like more than one usually appears in a tweet anyway.

**Comment by mroth** [comment]
Could be interesting.  The performance characteristics means it might be something to do server-side with a lua script, but I think it could be possible.  What is the set size here for a count: would it be `N^2` buckets where `N` is the number of distinct emoji glyphs?

**Comment by nowherenearithaca** [comment]
yes it seems to be N^2, which would be about 900^2... but there are a lot more emoji still to be included.  This might be an unrealistic thing to do, especially when there is not a clear thing that is going to be done with it.
